jf_agent/agent_logging.py

```diff
@@ -163,17 +191,28 @@ def handle(self, record: Union[logging.LogRecord, int]) -> None:
 
         if record != self._sentinel:
             msg = _standardize_log_msg(self.format(record))
-            self.messages_to_send.append(
-                {
-                    'message': msg,
-                    'timestamp': int(
+
+            # This logic is required to handle the different way that the timestamp attribute
```

Comment:
Q: Is there anything we could switch on here to determine which of these paths to take?

Reply:
Simplified this to always use `.created`